composer.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "phase2/svgo-php",
+  "description": "This is a PHP port of the SVGO [ https://github.com/svg/svgo ] library to optimize SVG vector graphic files.",
+  "require": {
```

Comment:
This is a bit light on typical details. See https://getcomposer.org/doc/04-schema.md for the formal spec. Look around at packages for typical stuff to include.